plugins/package-managers/maven/src/main/kotlin/Maven.kt

```diff
@@ -78,7 +78,7 @@ class Maven(
     }
 
     private inner class LocalProjectWorkspaceReader : WorkspaceReader {
-        private val workspaceRepository = WorkspaceRepository()
+        private val workspaceRepository = WorkspaceRepository("maven-cache")
```

Comment:
Consider: `"maven/remote-artifacts"` to avoid the term `cache in the directory name, because it is already under `cache/analyzer`.